docs/contributor_guide/development_process.rst

```diff
@@ -91,7 +91,38 @@ script which runs :code:`pip install` on all three of the above files.
 This script will also optionally pin the requirements to any lower bound specified (by changing any
 occurrences of :code:`>=` to :code:`==` in each file).
 
+.. _contributing_pull_requests:
 
+Contributing a pull request
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Follow the steps below to create a pull request.
+
+1. Get a `GitHub account <https://github.com/>`_
+
+2. Install `git <https://git-scm.com/book/en/v2/Getting-Started-Installing-Git>`_
+
+3. Look at Fairlearn's issues on GitHub, specifically the ones marked `"help wanted" <https://github.com/fairlearn/fairlearn/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22>`_. Within this category we've marked issues with labels:
+
+   - `"good first issue" <https://github.com/fairlearn/fairlearn/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22+label%3A%22good+first+issue%22>`_ means issues are a good fit for first time contributors including people with no prior experience with coding or GitHub. This is an excellent way to get started!
+   
+   - `"easy" <https://github.com/fairlearn/fairlearn/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22+label%3A%22easy%22+>`_ means the issue is somewhat harder than a "good first issue", but still quite doable if you have prior experience or you're willing to spend a little bit of time.
+   
+   - Neither of the two above: this means issues haven't been scoped out properly yet or are more difficult and likely won't be doable within a sprint. If you're still interested just let us know and we'll figure out a way! Once you find an issue you're interested in comment on the issue with any questions you may have and let us know that you'd like to do it. This helps us avoid duplication and we can help you quicker.
+
+4. Whenever questions come up don't hesitate to reach out (see the icons in the top menu). We're here to help!
```

Comment:
I think it would make sense to point to the communication channels section here. 

This makes me wonder: should discord be an icon at the top? Doesn't need to be part of this PR, of course, but if you agree we should perhaps open another issue for that.

Reply:
Added a reference to the communication section. I think adding discord as an icon at the top makes sense, as it's a direct way to get in contact with everyone

Reply:
Cool I will open up another issue for that and will resolve this comment!